inspire/modules/workflows/tasks/submission.py

```diff
@@ -287,3 +288,76 @@ def add_files_to_task_results(obj, eng):
         obj.add_task_result(file_obj.name,
                             fileinfo,
                             "workflows/results/files.html")
+
+
+def create_curation_ticket(template, queue="Test", ticket_id_key="ticket_id"):
+    """Create a ticket for the curation.
+
+    Creates the ticket in the HEP_curation queue and stores the ticket ID
+    in the extra_data key specified in ticket_id_key."""
+    @wraps(create_curation_ticket)
+    def _create_curation_ticket(obj, eng):
+        if not obj.extra_data.get("core"):
+            obj.log.info("Curation ticket creation skipped.")
+            return
+        from invenio.modules.access.control import acc_get_user_email
+        from inspire.utils.tickets import get_instance
+        from flask import render_template
+
+        d = Deposition(obj)
+        email = acc_get_user_email(obj.id_user)
+        rt_queue = "Test"
```

Comment:
It's so that I don't pile up the HEP_Curation queue with tests... It's still a WIP ;)